src/Middleware/Diagnostics/src/DeveloperExceptionPage/Views/ErrorPage.cshtml

```diff
@@ -249,6 +253,78 @@
                 <p>@Resources.ErrorPageHtml_NoHeaderData</p>
             }
         </div>
+
+        <div id="routingpage" class="page">
```

Comment:
Should we show this when there is no endpoint info? What does it look like when you're using non-endpoint routing?

Reply:
It will say “No endpoint”. That is what the other tabs do when there is no data